CHANGELOG.md

```diff
@@ -1,13 +1,20 @@
 # Change Log for hipBLAS
 
-## (Unreleased) hipBLAS 1.1.0
+## (Unreleased) hipBLAS 2.0.0 for ROCm 6.0.0
+### Deprecated
+- hipblasDatatype_t is removed and replaced with hipDataType
+- hipblasComplex and hipblasDoubleComplex are removed and replaced with hipComplex and hipDoubleComplex
```

Comment:
This wording is not clear to me.  Either it is deprecated in this version or it is removed.  These deprecated ones should be "are to be removed and can be ...".  Would be nice to detail how to use the new/old API for these cases in a bit more detail.

Reply:
Thanks, improved wording to for deprecated datatypes.